shared/challenge_builder.py

```diff
@@ -92,7 +98,8 @@ def queryElementsRaw(self, overpass_query):
     def queryElementsAsGeoJSON(self, overpass_query, forceGeomType=None):
         rawElements = self.queryElementsRaw(overpass_query)
         geometriesAndProperties = [(self.geoJSONGeometryFromOverpassElement(element, forceGeomType)) for element in rawElements]
-        return [geojson.Feature(geometry=g, properties=p) for g,p in geometriesAndProperties]
+        featureList = [geojson.Feature(geometry=g, properties=p) for g,p in geometriesAndProperties]
+        return featureList
```

Comment:
Why adding another variable instead of the direct return as before?

Reply:
I don't get it but if you want to go this way, at least use a [map lambda function](https://www.geeksforgeeks.org/python-map-function/).

Reply:
Should be good now